lib/progress.ts

```diff
@@ -73,16 +75,19 @@ const getProgressStream = function (total, onState) {
  * 		stream.on 'progress', (state) ->
  * 			console.log(state)
  */
-export function estimate(requestAsync, isBrowser) {
-	return async function (options) {
+export function estimate(
+	requestAsync?: ReturnType<typeof getRequestAsync>,
+	isBrowser?: boolean,
+) {
+	return async function (options: BalenaRequestOptions) {
 		if (requestAsync == null) {
 			requestAsync = utils.getRequestAsync();
 		}
```

Comment:
I think it's worth converting all these nullish defaults to es6 defaults instead since we're doing a breaking

Reply:
Will do.
Did you spot any other place we could do that, since that was the only one I could find where es6 defaults could be used.